sled-agent/src/services.rs

```diff
@@ -540,6 +571,16 @@ struct SledAgentInfo {
     rack_network_config: Option<RackNetworkConfig>,
 }
 
+pub(crate) enum TimeSyncConfig {
```

Comment:
This is *slightly* more complicated than a bool to assist with testing. I need an option that fails unconditionally to avoid mocking out all the NTP calls.